py3dtilers/IfcTiler/IfcTiler.py

```diff
@@ -60,14 +65,20 @@ def main():
 
     this function creates an ifc tileset handling one ifc classe per tiles
     """
+    logging.basicConfig(filename='ifctiler.log', level=logging.INFO,filemode="w")
+    start_time = time.time()
+    logging.info('Started')
     ifc_tiler = IfcTiler()
     ifc_tiler.parse_command_line()
     args = ifc_tiler.args
+    args.offset = np.array(args.offset,dtype=np.float32)
```

Comment:
Maybe this line should be in [tiler.py](https://github.com/VCityTeam/py3dtilers/blob/master/py3dtilers/Common/tiler.py#L94) ?